en/user-guide/track_lines_polygons.rst

```diff
@@ -0,0 +1,51 @@
+.. _track_linges_polygons:
+
+Tracking of lines and polygons
+==============================
+
+You have the possibility to do tracking on lines and polygons.
+
+The option is available when GPS is enabled. 
+Tracking can be started over the legend - long press on the layer.
+
+.. container:: clearer text-center
+
+    .. image:: /images/track_lines_started.jpeg
+       :width: 640px
+       :alt: track_lines_started
+
+
+The time interval and the minimum distance can be entered individually to define when the vertex need to be set.
+
+.. container:: clearer text-center
+
+    .. image:: /images/track_lines_tracker_settings.jpeg
+       :width: 640px
+       :alt: tracker_settings
+
+
+There can be several trackings active on several layers. 
+But only one tracking session per layer can be active at the time. 
```

Comment:
What is the difference between a "tracking (on a layer)" and a "tracking session"?
I think this needs some explanation somewhere.

Reply:
As I used it in the PR comment I thought tracking starts when the track is recorded and a tracking session starts earlier, when the user enters the tracking properties - but I guess we do not need to make a difference here in the documentation, so I would only use one. I like tracking but tracking session is maybe more understandable.

Reply:
@m-kuhn do you prefer tracking or tracking session?

Reply:
I like tracking session but no strong opinion

Reply:
So lets take tracking session. I think it's easier to understand for the user.